File: piramid.lua

function create_piramid( rows )
	spaces = rows
	for i = 0, rows * 2, 1 do
		if( 0 == (i % 2) ) then
			-- esta mierda no tiene continue!! :'( *llora*
			-- y yo no voy a usar goto ni que me paguen
			_spaces = ''
			_asterisks = ''
			for j = 0, spaces, 1 do
				_spaces = _spaces .. ' '
			end
			for n = 0, i, 1 do
				_asterisks = _asterisks .. '*'
			end
			print( _spaces .. _asterisks )
			spaces = spaces - 1
		end
	end
end

create_piramid( 30 )

File: piramid.java
public class piramid {
	private static void create_piramid( int rows ) {
		int spaces = rows;
		for( int i = 0; i < rows * 2; i++ ) {
			if( 0 == ( i % 2 ) )
				continue;
			for( int j = 0; j < spaces; j++ ) {
				System.out.print(" ");
			}
			for( int n = 0; n < i; n++ ) {
				System.out.print("*");
			}
			spaces--;
			System.out.println("");
		}
	}
	public static void main( String[] args ) {
		create_piramid( 30 );
	}
}